app/controllers/application_controller.rb

```diff
@@ -2,4 +2,11 @@ class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
+
+  def authorize_super_admin!
```

Comment:
I think this should be a private method as per https://github.com/empowerhack/DrawMyLife-Service/pull/153/files#diff-b49f873ae722d049d381a3fed3b3b5c5L61